projects/experimental/directives/ripple/ripple.style.less

```diff
@@ -0,0 +1,39 @@
+@keyframes tuiRippleOn {
+    from {
+        transform: scale(0);
+        opacity: 0.12;
+    }
+
+    to {
+        transform: none;
+        opacity: 0.12;
+    }
+}
+
+@keyframes tuiRippleOff {
+    from {
+        transform: none;
+        opacity: 0.12;
+    }
+
+    to {
+        transform: none;
+        opacity: 0;
+    }
+}
+
+.tui-ripple-container {
+    position: relative;
```

Comment:
I'm not sure that it won't break something. How about `position: absolute/fixed` elements?